bookshelves.md

```diff
@@ -8,39 +8,60 @@ The Free Ebook Foundation (FEF) has been helping Project Gutenberg (PG) renovate
 
 “Bookshelves” are lists of PG books that help users navigate PG. See them [here](https://www.gutenberg.org/wiki/Category:Bookshelf).
 
-
 ## Problem
 
 - System for creating and updating bookshelves is failing
-    - PHP/wiki interface
-    - Loaded into production Postgres db by scraping wiki!
-    - Few people maintaining it.
-    - No way to create personal bookshelves or reading lists
-    - Recurring security issues with PHP
-
+  - PHP/wiki interface
+  - Loaded into production Postgres db by scraping wiki!
+  - Few people maintaining it.
+  - No way to create personal bookshelves or reading lists
+  - Recurring security issues with PHP
 
 ## Proposed Solution
 
 - Registrationless Bookshelf Builder
-    - HTML5 local storage + javascript
-    - Button on Item page
-    - Import/export lists for sharing
-
-- Bookshelf Management App
-    - Python/Flask or Python/CherryPy with Postgres backend
-    - Managers can make new bookshelves and categories
-    - Moderators can add/remove items from Builder Bookshelves
 
+  - HTML5 local storage + javascript
+  - Button on Item page
+  - Import/export lists for sharing
 
+- Bookshelf Management App
+  - Python/Flask or Python/CherryPy with Postgres backend
```

Comment:
Python/Django is also a reasonable choice. You can run Flask in CherryPy as well. Could do it with Node, but there's not much experience with JS apps on the PG IT team.

Reply:
Personally I am most comfortable with Node of those options, but that does not mean it is the best tool for the job. I think we would make the decision on this as we review the existing code and detail out some of the user stories. The decision would be finalized by the implementation plan in the updated timeline